packages/react-composites/src/composites/CallComposite/components/SidePane/usePeoplePane.tsx

```diff
@@ -322,18 +324,20 @@ export const usePeoplePane = (props: {
 
   const onRenderContent = useCallback((): JSX.Element => {
     return (
-      <>
+      <div key="asdfusepeoplepane">
```

Comment:
still some asdf's left (revert that this is a div too)

Reply:
My vscode was in a weird state where the commit with asdf removal did not work... It was very odd. 
I resolved it with a hard reset of vscode. Should be good now!